.github/workflows/ci.yml

```
@@ -12,8 +12,8 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3
-    - uses: volta-cli/action@v4
+    - uses: actions/checkout@v4.1.7
+    - uses: volta-cli/action@v4.1.1
```

What's the difference here, I thought `@v4` would always just use the latest on that version. Are you pinning it to a specific number to ensure nothing breaks? Or is my assumption incorrect, and `@v4` actually means `@v4.0.0`?
```suggestion
    - uses: actions/checkout@v4
    - uses: volta-cli/action@v4
```